README.adoc

```diff
@@ -37,6 +37,7 @@ This is the server component of the AeroGear Mobile Security Service. It is a RE
 |https://github.com/golang/go/wiki/SettingGOPATH[Ensure the $GOPATH environment variable is set]
 |https://docs.docker.com/compose/install/[Install the dep package manager]
 |https://docs.docker.com/compose/install/[Install Docker and Docker Compose]
+|https://www.postgresql.org/[PostgreSQL] database
```

Comment:
This pre-req is the only one that doesn't start with a verb, what does the user need to do with Postgres?

Reply:
@laurafitzgerald without the Postgres is not possible to start the server.

Reply:
@camilamacedo86 Agreed but the question is more around the way it's asked compared to the others.
You say 
`Install Docker and Docker Compose`
and 
`Install the dep package manager`
and then 
`Postgres`
Should the user install it or run it maybe?

Reply:
maybe it isn't a pre-req, given that the install process is discussed below?

Reply:
@finp Yes I agree that we should have it as _either_ a pre-req or list the install process.